docs/concepts/distributed_trust.md

```diff
@@ -8,53 +8,29 @@ This feature is currently a preview on Peregrine and is therefore not available
 :::
 
 Attesters can be individuals that attest to the validity of claims made by claimers.
-But in many cases multiple attesters group together to build up a trust in a brand.
+But in many cases multiple attesters group together to build up trust in a brand.
 Verifier don't need to trust individual attesters but trust the brand, which in return ensures that all attesters working for this brand are credible.
-
 Such a brand can be organized in many different ways.
-Today the most common way to organize and build a brand is to found a company.
-In the future their are going to be more decentralized solutions like Decentralized Autonomous Organizations (DAOs) or more specifically Virtual Credential Organizations (VCOs).
-
-If such a collective has given the attester the right to issue attestations, the attester can decide to create an attestation in the name of the collective.
-The attester is still the only issuer of the attestation, but the collective has power over this attestation.
-If the attester later leaves the collective, depending on the collective they may not be able to revoke or remove the attestation.
-Only the collective can then manage the attestation.
-But even if the attester is still part of the collective, the collective has power over the attestation.
-
-<center>
-
-![](/img/concepts/distributed_trust/attester-collective.svg)
-
-</center>
-
-The image above shows an attester collective which consists of 3 attesters (Attester 1, 2 and 3).
-Attester 4 is not part of the collective.
-Attester 1 create an attestation for the collective.
-Attester 2 and 3 can now revoke or remove this attestations.
-Attester 4 cannot create attestations for the collective and can also not revoke or remove any attestation that belong to the collective.
-
-Attester collectives are a powerful construct.
-But how can Attester 4 become part of the collective?
-How does the collective handle malicious members?
-Maybe Attester 2 should have the power to revoke but not to remove an attestation?
+The KILT protocol provides mechanisms to form such brands on the blockchain.
 
-All of these questions are solved by the specific type of collective.
-At the moment there is only a single type of collective available, the Delegation Hierarchy.
-A future attester collective type is the VCO which is more powerful than a simple Delegation Hierarchy and provides more flexible tools.
+There are two ways for attesters to group up and build an organization.
+The first one are Delegation Hierarchies, which provide a very basic hierarchical organisation.
+Delegation Hierarchies are available on our test network Peregrine.
+The second option are Virtual Credential Organizations which are not yet implemented and currently.
+Virtual Credential Organizations will be more flexible than Delegation Hierarchies and accommodate a wider range of use cases.
```

Comment:
I would not call it wider, but different. For instance VCOs are unable to represent hierarchies at all, which is something that some use cases might require. I agree on the flexibility thought.

Reply:
[8074d7f](https://github.com/KILTprotocol/docs/pull/88/commits/8074d7fb0d6a3a6892fefb13609766d6929fa3bd)